runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -328,13 +347,21 @@ public void onChatMessage(ChatMessage event)
 			}
 		}
 
+		if (config.screenshotCcKick() && chatMessage.startsWith("Your request to kick/ban this user was successful."))
+		{
+			final String nameText = kickPlayerName == null ? "" : kickPlayerName;
+			String fileName = "Kick " + nameText;
```

Comment:
Not sure if it matters here the filename will be `Kick ` if there is no name.

Reply:
There should always be a name since we’re grabbing the kicked player’s name in a callback immediately following the kick event

Reply:
There probably should, but the line above this has logic to handle if there isn't, so it should be consistent one way or the other (maybe early return if kicked name is null?)